core.go

```diff
@@ -18,6 +18,44 @@ const (
 	zapSentryScopeKey = "_zapsentry_scope_"
 )
 
+var (
+	defaultFrameFilters = FrameFilters{
+		SkipModulePrefixFrameFilter("github.com/TheZeroSlave/zapsentry"),
+		SkipFunctionPrefixFrameFilter("go.uber.org/zap"),
+	}
+)
+
+type (
+	FrameFilter interface {
+		SkipFrame(f sentry.Frame) bool
+	}
+	FrameFilters                  []FrameFilter
+	FrameFilterFunc               func(f sentry.Frame) bool
+	SkipModulePrefixFrameFilter   string
+	SkipFunctionPrefixFrameFilter string
+)
+
+func (f FrameFilterFunc) SkipFrame(frame sentry.Frame) bool {
+	return f(frame)
+}
+
+func (f SkipModulePrefixFrameFilter) SkipFrame(frame sentry.Frame) bool {
+	return strings.HasPrefix(frame.Module, string(f))
+}
+
+func (f SkipFunctionPrefixFrameFilter) SkipFrame(frame sentry.Frame) bool {
+	return strings.HasPrefix(frame.Function, string(f))
+}
+
+func (ff FrameFilters) AnyMatch(frame sentry.Frame) bool {
```

Comment:
I think that a better design would be to have `FrameFilters` implement the `FrameFIlter` interface.

Reply:
right, I'll update this